internal/operator-controller/rukpak/util/testing/bundlefs.go

```diff
@@ -0,0 +1,64 @@
+package testing
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+	"testing/fstest"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/yaml"
+)
+
+const (
+	BundlePathAnnotations = "metadata/annotations.yaml"
+	BundlePathProperties  = "metadata/properties.yaml"
+	BundlePathManifests   = "manifests"
+	BundlePathCSV         = BundlePathManifests + "/csv.yaml"
+)
+
+func NewBundleFS() fstest.MapFS {
```

Comment:
What's the benefit to moving this out of the bundle/source/source_test.go? I'm not seeing it used anywhere else. It just feels as though it's making the PR bigger without benefit?

Reply:
It's used in the boxcutter applier test [[e.g.](https://github.com/operator-framework/operator-controller/pull/1946/files#diff-81cef2cf392f91d8e015a9d1b8cbf139cf52cb8b42ecb2b212d0fde783addad6R50)]